scale_tests/framework/concurrent_resource_creator.py

```diff
@@ -104,6 +111,9 @@ def _create_deployment(self, blueprint_id):
                                        deployment_id=uuid.uuid4().hex,
                                        inputs=self.blueprint_example.inputs)
 
+        if self.wait_after_action:
```

Comment:
You could probably just make the default value `0`. Then it would have 2 benefits:
1. Remove this `if` (you just sleep 0 seconds)
2. Make it more obvious what `wait_after_action` is supposed to be (I thought at first that it's a function you perform after the action, or maybe a boolean that decides whether to wait at all or not).